thunder/dynamo/compiler.py

```diff
@@ -208,16 +208,80 @@ def save_reproducer_to_folder(
                 )
 
 
-def thunderfx(fn: Callable, /, **kwargs) -> Callable:
-    """Compiles a callable (function or model) by using Thunder as the backend of :func:`torch.compile`
+# We return this object instead of just the raw `compiled` Callable so that
+# we have a place to hang the `last_*traces` properties.
+class ThunderfxCompiledObject:
+    """A wrapper around the result of :func:`~thunder.dynamo.thunderfx` compilation.
+
+    This object wraps the function compiled with :func:`torch.compile` using Thunder as a backend
+    (see :class:`~thunder.dynamo.ThunderCompiler`).
+    It provides access to Thunder traces generated during execution, while maintaining the
+    callable interface of the original compiled function.
+
+    Note: This is the return type of ``thunderfx``, not :func:`thunder.jit`.
+    """
+
+    def __init__(self, be, func: Callable):
```

Comment:
It would be nice to rename `be` as `backend`. It looks a bit confusing in the generated doc otherwise.

![image](https://github.com/user-attachments/assets/49a30dc6-a296-4d42-96a0-c6702186bcd7)